docs/querying/basics.md

```diff
@@ -81,6 +81,16 @@ Examples:
     0x8f
     -Inf
     NaN
+	
+
+As of version 2.52, float values can also be represented using the syntax of durations, where the duration literal is converted into a float value corresponding to the number of seconds the duration literal represents.
```

Comment:
Let's be super precise with the terminology here. A float value is different from a float literal, and duration literals are called "time durations" in this doc. So this should read:

"As of version 2.53, float literals can also be represented using the syntax of [time durations](#time-durations), where the time duration is converted into a float value corresponding to the number of seconds the time duration represents. This is an experimental feature and might still change."

Note that I changed 2.52 to 2.53, because the 2.52 RC is already out. I also added a note about the feature being experimental (copied from your text below for time durations).